lib/entities/generate_name.dart

```diff
@@ -20,5 +20,6 @@ Future<String> generateName() async {
   final chosenNoun = nouns[randomThing.nextInt(nouns.length)];
   final returnString =
       chosenAdjective.capitalized() + ' ' + chosenNoun.capitalized();
+    
```

Comment:
Why it appeared here ? Do not modify parts which are not effected by your implementation